### ISSUE TITLE
GPT: Create an Ansible Playbook and push to repo

### ISSUE THREAD

**Issue by serenamarie125** [janus-idp/software-templates#128]
## Goal
_Details to follow_

### What problem does this solve?
_Details to follow_
 
### Use cases
_Details to follow_

### Acceptance criteria
_Details to follow_

## Issues in Epic
- https://github.com/janus-idp/software-templates/issues/147
- https://github.com/janus-idp/software-templates/issues/173 [ENG]
- https://github.com/janus-idp/software-templates/issues/157 [DOC]
- https://github.com/janus-idp/software-templates/issues/156 [QE]

**Comment by rtaniwa** [comment]
@jasperchui assigning to you as the engineering work is being done by Ryan Cook. Once the PR is approved and merged, this will satisfy the engineering work required for this one and we would just need to track Doc and QE work.

**Comment by kadel** [comment]
As discussed during the triage call, this can be shipped independently from Milestone 3 or Milestone 4.

There is also the outstanding question of how this is different from #127 (what is the difference between launching and creating an ansible playbook in the context of the Backstage template?)

**Comment by jasperchui** [comment]
No longer needed according to https://github.com/janus-idp/software-templates/issues/193
